main.py

```diff
@@ -57,35 +54,35 @@ def frames_to_tensor(frames):
 state = np.zeros((1,512))
 desire = np.zeros((1,8))
 
+cap = cv2.VideoCapture(camerafile)
```

Comment:
That is duplicate as it is already defined in line 24.

Reply:
Actually this was done to play the video from start because VideoCapture ended while reading frames into 'imgs' array.

Reply:
Alright, merged